py/desispec/io/spectra.py

```diff
@@ -212,11 +233,20 @@ def read_spectra(infile, single=False):
         raise IOError("{} is not a file".format(infile))
 
     t0 = time.time()
-    hdus = fitsio.FITS(infile, mode='r')
+    hdus = fitsio.FITS(infile, mode="r")
     nhdu = len(hdus)
+    ahdus = fits.open(infile)
```

Comment:
Let's add a comment about why we need both `hdus` and `ahdus` for the same file.  I think the answer is that fitsio is more efficient overall but only supports reading rows of tables but not rows of 2D arrays, so we need astropy `ahdus` when slicing rows of FLUX, IVAR, etc.

Reply:
comment added